porter/v1/cluster_control_plane.proto

```diff
@@ -165,4 +216,16 @@ service ClusterControlPlaneService {
   // AssumeRoleCredentials should be used vary sparingly, and ONLY for replacing AWS Integrations which have no workaround on the Porter API.
   // This endpoint returns temporary AWS credentials for a given AWS Account ID, and should not be expanded further to allow specifc role selection without being tied to a project and cluster
   rpc AssumeRoleCredentials(AssumeRoleCredentialsRequest) returns (AssumeRoleCredentialsResponse) {}
+
+  // TokenForRegistry returns a docker-compatible token for accessing a given registry
+  rpc TokenForRegistry(TokenForRegistryRequest) returns (TokenForRegistryResponse) {}
+
+  // DockerConfigFileForRegistry returns a docker-compatible token for accessing a given registry
+  rpc DockerConfigFileForRegistry(DockerConfigFileForRegistryRequest) returns (DockerConfigFileForRegistryResponse) {}
```

Comment:
Is this a Docker config as in the literal `docker config ls` config file, or a daemon.json, or a config.json?

Reply:
It's a config.json file (docker calls it a ConfigFile in their go code)